docs/database.rst

```diff
@@ -0,0 +1,40 @@
+rotki Database
+##############
+
+rotki uses two different sqlite databases, one with information about assets (global.db) and one with user information (rotkehlchen.db).
+The latter is encrypted using an extension called `SQLCipher <https://github.com/sqlcipher/sqlcipher>`__
+that provides transparent 256-bit AES full database encryption.
+
+Database Location
+=================
+
+The default database locations are:
+
+- **Linux**: ``~/.share/local/rotki/data``
+- **OSX**: ``??``
+- **Windows**: ``??``
+
+Exploring the database
+**********************
+
+To open the user database you can use `sqlitebrowser <https://sqlitebrowser.org/>`__.
+It supports sqlcipher and will ask for the password used to decrypt the database.
+
+If you prefer the command line instead, you can use the ``sqlcipher`` cli tool. Note: using the ``sqlite3`` cli will not work!
+
+Note to Debian and Ubuntu users: we are using SQLCipher encryption v4, therefore a recent version of sqlcipher is required. Unfortunately
+the version available in the distribution repositories are too old and won't let you open the database using ``sqlitebrowser`` nor ``sqlcipher``.
+On top of that, the ``sqlitebrowser`` version provided is not compiled with sqlcipher support.
+As a workaround, you can use `this PPA <https://launchpad.net/~linuxgndu/+archive/ubuntu/sqlcipher>`__ to
+install more recent versions of both packages and you can also recompile sqlitebrowser with sqlcipher support following this
+`stackoverflow thread <https://stackoverflow.com/questions/48105035/sqlite-browser-without-sqlcipher-support-in-ubuntu>`__.
+
+When using sqlcipher, you need to specify the password to decrypt the database entering ``PRAGMA key='your-secret-key';`` right after opening the database.
+
+Migrations
+==========
+
+Database migrations are needed when changes in the schema happen. rotki checks a setting in the database with the version and
+just executes sequentially a check against the version to verify if the migration needs to happen or not.
```

Comment:
Those migrations happen for the user db and in develop for globaldb. Also we removed some old migrations so maybe you can also link that part in the documentation

Reply:
I don't really understand what you mean. Can you propose a new text?